ui/pages/desktop-error/render-desktop-error.js

```diff
@@ -43,6 +45,12 @@ export function renderDesktopError({
   };
 
   const renderHeader = (text) => {
+    if (typeof trackEvent === 'function') {
+      trackEvent({
+        category: EVENT.CATEGORIES.ERROR,
+        event: `Error: ${text}`,
+      });
```

Comment:
Why are we tracking errors in segment?

Reply:
The goal here was to measure the errors while using the desktop app as a companion but you are right, It is already collected by Sentry and if necessary in Segment, it can be tracked by using the `trackPage` method. I will remove it and push the changes.